### PR TITLE
Update test_l2_basic.py

### PR DESCRIPTION
**Pull request by bohdanblavatnyi**
Converted L2MacMoveTestI() from PTF into SC

### CODE REVIEW

File: tests/test_l2_basic.py [hunk 1]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
 
     finally:
         npu.flush_fdb_entries(["SAI_FDB_FLUSH_ATTR_BV_ID", npu.default_vlan_oid])
+def test_l2_mac_move_1(npu, dataplane):
```
Comment (vitaliy-senchyshyn-plv) [inline tests/test_l2_basic.py:509]: Please add two empty lines before test function.

**Review by vitaliy-senchyshyn-plv — APPROVED** [review]

File: tests/test_l2_basic.py [hunk 2]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
...
+    1. Create VLAN 100 and associate 3 untagged ports (port 1,2,3) as the member port of VLAN 100.
+    2. Set attribute value of "Port VLAN ID 100" for the ports 1, 2, 3
+    3. Send packet from port 1 with src_mac=MAC1 and dst_mac= MAC2
+    4. Send packet from port 2 with src_mac=MAC2 and dst_mac= MAC1.
+    5. After MAC learning, repeat step 3 and verify that packet from port 1 is only forwarded to port 2 and not to port 3.
+    6. Send packet (src_mac=MAC2 and dst_mac= MAC1) on port 3
+    7. Send packet (src_mac=MAC1 and dst_mac=MAC2) from port 1.
+    8. Clean up by remove the vlan members and the VLAN from the database.
+    '''
+    print('L2 Mac Move test on Access ports 1,2 and 3')
```
Comment (plvisiondevs) [inline tests/test_l2_basic.py:530]: Will not be shown in pytest environment. Please remove.

File: tests/test_l2_basic.py [hunk 3]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
...
+                                     eth_src=macs[1],
+                                     ip_dst='192.168.0.1',
+                                     ip_src='192.168.0.2',
+                                     ip_id=105,
+                                     ip_ttl=64)
+                  
+            send_packet(dataplane, 0, pkt)
+            verify_packets(dataplane, pkt, [1])
+
+            ### Sending packet from Port1
```
Comment (plvisiondevs) [inline tests/test_l2_basic.py:565]: Please remove this excessive comment here and below and keep just send/verify same as above
```
send_packet(..)
verify_packets(..)
```

File: tests/test_l2_basic.py [hunk 4]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
...
+                                     ip_ttl=64)
+
+            pkt2 = simple_tcp_packet(eth_dst=macs[0],
+                                     eth_src=macs[1],
+                                     ip_dst='192.168.0.1',
+                                     ip_src='192.168.0.2',
+                                     ip_id=105,
+                                     ip_ttl=64)
+                  
+            send_packet(dataplane, 0, pkt)
```
Comment (plvisiondevs) [inline tests/test_l2_basic.py:562]: No `pkt` defined. Will cuase runtime failure with traffic

File: tests/test_l2_basic.py [hunk 5]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
...
+
+            ### Send packet (src_mac=MAC1 and dst_mac=MAC2) from port 1
+            send_packet(dataplane, 0, pkt1)
+
+            ### verify the packet @ Port3 & no packet @ Port2
+            verify_packets(dataplane, pkt1, [2])
+            verify_no_packet(dataplane, pkt1, 1)
+   
+    finally:
+        ### Assign ports into default vlan and remove vlan membership from ports
```
Comment (plvisiondevs) [inline tests/test_l2_basic.py:606]: Please add `npu.flush_fdb_entries(["SAI_FDB_FLUSH_ATTR_BV_ID", vlan_oid])`

File: tests/test_l2_basic.py [hunk 6]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
...
+        for idx in range(max_port):
+            npu.remove(vlan_mbr_oids[idx])
+            npu.create_vlan_member(npu.default_vlan_oid, npu.dot1q_bp_oids[idx], "SAI_VLAN_TAGGING_MODE_UNTAGGED")
+            npu.set(npu.port_oids[idx], ["SAI_PORT_ATTR_PORT_VLAN_ID", npu.default_vlan_id])
+        
+        ### remove valn
```
Comment (plvisiondevs) [inline tests/test_l2_basic.py:612]: Please remove excessive comment

File: tests/test_l2_basic.py [hunk 7]
```diff
@@ -506,3 +506,108 @@ def test_fdb_bulk_remove(npu, dataplane):
...
+    7. Send packet (src_mac=MAC1 and dst_mac=MAC2) from port 1.
+    8. Clean up by remove the vlan members and the VLAN from the database.
+    '''
+    print('L2 Mac Move test on Access ports 1,2 and 3')
+
+    vlan_id = "100"
+    macs = ['00:11:11:11:11:11', '00:22:22:22:22:22']
+    max_port = 3
+    vlan_mbr_oids = []
+    ### create vlan
```
Comment (plvisiondevs) [inline tests/test_l2_basic.py:536]: Please remove excessive comment

**Comment by plvisiondevs** [comment]
Good job! Thank you.
The PR will be approved and merged once the review comments are resolved.

**Review by plvisiondevs — APPROVED** [review]

**Pull request merged**